README.md

```diff
@@ -2,19 +2,63 @@
 
 A PHP package for mapping files from one location to another. It is used by the [youwe/composer-file-installer](https://github.com/YouweGit/composer-file-installer) package to move installed files according to the location mapping.
 
+## Mapping notation
+
+The mapping can contain `{source,destination}` placeholders, which will be replaced with the source or destination part 
+relatively. A mapping string can contain multiple `{source,destination}` placeholders.
+
+Examples mappings:
+
+| Mapping string                   | Translates to source path  | Translates to destination path |
+|----------------------------------|----------------------------|--------------------------------|
+| `file.php`                       | `file.php`                 | `file.php`                     |
+| `{dot,.}gitignore`               | `dotgitignore`             | `.gitignore`                   |
+| `{default/,}config.yaml{.dist,}` | `default/config.yaml.dist` | `config.yaml`                  |
```

Comment:
I would like to suggest to use the `|` character. Not only is this a very common character in substitution formats it is also a character which you will never find in file names as it is reserved in both *nix and Windows based systems.